### ISSUE TITLE
[Enhancement] Call for Contributions: Expand Our Machine Learning Project with Datasets and Code

### ISSUE THREAD

**Issue by 1chooo** [1chooo/simple-ai#66]
We have currently completed the initial functionality of this project. In order to broaden the scope of our project and make our goals even more ambitious, we are looking to collaborate with the open-source community. We hope that everyone can contribute by providing more and more machine learning examples. All kinds of datasets are highly welcomed. Developers who wish to contribute can share links to datasets and place pre-trained code in the /materials directory. Your contributions are greatly appreciated.

**Comment by DmitriiPodlev** [comment]
Hi! I can fork new project to your Refinaid/Materials folder. Do you have any suggestions for the project's topic? Or what machine learning models do you want to see in project? Are there any requirements?

**Comment by 1chooo** [comment]
Hello! You can review items #5 and #9, which contain lists of datasets and ML models we plan to incorporate into our project. Please make a note of the dataset you'd like to contribute, and I'll handle the project structure. Thank you for your fantastic work!

**Comment by DmitriiPodlev** [comment]
Ok. I can take Titanic's dataset and build models Decision Tree, Random Forest, Linear and Boosting. Is it right for you? I also can find another dataset in Kaggle and build these models for new dataset.

**Comment by 1chooo** [comment]
Hey @DmitriiPodlev, that's right! We are very expected to your amazing jobs!

**Comment by DmitriiPodlev** [comment]
Ok! Sorry, I accidentally deleted myself from Assignees. Please, appoint me agaun) I can start and push code within a few days. Which of branchs can I use? Do I need create new branch?

**Comment by 1chooo** [comment]
Certainly, you can fork this repository and review the "Collaboration Guidelines." Simply create pull requests (PRs), and if there are no issues during my review, I will merge them!

**Comment by DmitriiPodlev** [comment]
Ok! I will start it)

**Comment by DmitriiPodlev** [comment]
Hello! I create new branch with my job. But I can't publish my branch. Can you give me please rights to publish branch. Then I will create pull request

**Comment by 1chooo** [comment]
You have to fork into your workspace first, and do the amazing jobs under your workspace/simple-ai, then you can add the commit, finally you can launch the pull requests, and I will review!

You can view more details in collaboration guidelines!

If still encountered any problems, please feel free to let me know.
